amundsen_application/static/js/components/common/ResourceList/styles.scss

```diff
@@ -23,7 +23,8 @@
   .empty-message {
     display: flex;
     justify-content: center;
-    margin-top: $spacer-4;
+    margin: $spacer-4 $spacer-4 0;
```

Comment:
Cleanup: These styles just make the error text look nicer if it gets really long on a screen that isn't very wide.